bin/configurer/utils/idam-create-service.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -eu
+
+dir=$(dirname ${0})
+
+LABEL="${1:-ccd_gateway}"
+SELF_REG="${2:-false}"
+CLIENT_ID="${3:-ccd_gateway}"
+CLIENT_SECRET="${4:-ccd_gateway_secret}"
+REDIRECT_URLS="${5:-[\"http://localhost:3451/oauth2redirect\"]}"
+
+apiToken=$(${dir}/idam-authenticate.sh "${IDAM_ADMIN_USER}" "${IDAM_ADMIN_PASSWORD}")
+
+echo -e "\nCreating service with:\nLabel: ${LABEL}\nClient ID: ${CLIENT_ID}\nClient Secret: ${CLIENT_SECRET}\nRedirect URL: ${REDIRECT_URLS}\n"
+
+STATUS=$(curl -s -o /dev/null -w '%{http_code}' -X POST -H 'Content-Type: application/json' -H "Authorization: AdminApiAuthToken ${apiToken}" \
```

Comment:
To be honest I prefer longer versions of switches such as `-s`, `-o` or `-w`. It make it more readable IMO. Short versions mean sometimes different things in different tools :(

Reply:
Done.